MageOS/AsyncEventsAWS/etc/adminhtml/system.xml

```diff
@@ -2,32 +2,50 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
     <system>
-        <section id="async_events_aws" showInDefault="1" showInWebsite="1" showInStore="1">
+        <section id="async_events_aws" showInDefault="1" showInStore="1">
```

Comment:
Should we remove `showInStore="1"` too here and in the groups and fields? The config model doesn't allow to set a store, so we will always retrieve the global config values.
The other option would be to make everything scope specific so you can configure different credentials for different websites / stores. In that case, I'd prefer to restore `showInWebsite="1"`.

Reply:
I was actually debating this. Currently the configuration classes fetch from the global scope however they can be made to fetch from store specific scope (we have access to the store ID via a CloudEvent's source).

However we're instantiating the AWS clients as singleton which will pass one set of keys the first time and that's it.

Reply:
Also the reason why it's not website scoped is because subscriptions are scoped per website. The `store_id` in the DB is what we're using. Unless you know a way to get the default website by providing a store ID?

Reply:
I've removed `showInStore` for now. this is a possible future improvement :+1: